transport/src/main/java/io/netty/channel/local/LocalChannel.java

```diff
@@ -306,7 +322,7 @@ protected void doWrite(ChannelOutboundBuffer in) throws Exception {
         case BOUND:
             throw new NotYetConnectedException();
         case CLOSED:
-            throw new ClosedChannelException();
+            throw CLOSED_CHANNEL_EXCEPTION;
```

Comment:
Not that it isn't useful, just more consistent with other channels (see [AbstractChannel](https://github.com/netty/netty/blob/master/transport/src/main/java/io/netty/channel/AbstractChannel.java#L793)).